prow/pod-utils/gcs/upload.go

```diff
@@ -35,29 +35,45 @@ type UploadFunc func(obj *storage.ObjectHandle) error
 // Upload uploads all of the data in the
 // uploadTargets map to GCS in parallel. The map is
 // keyed on GCS path under the bucket
+// 3 attempts will be made to have an error free upload
+const uploadRetries = 3
 func Upload(bucket *storage.BucketHandle, uploadTargets map[string]UploadFunc) error {
+
```

Comment:
Can we remove all the extra empty lines added to this file?

Reply:
let me know if there are any more to clean up. I'm not sure the style to follow and I usually am pretty liberal with blank lines for my own readability.